pkg/operation/botanist/component/kubeapiserver/secrets.go

```diff
@@ -366,21 +348,6 @@ func (k *kubeAPIServer) reconcileSecretHAVPNSeedClient(ctx context.Context) (*co
 	return secret, nil
 }
 
-func (k *kubeAPIServer) reconcileSecretLegacyVPNSeedTLSAuth(ctx context.Context) (*corev1.Secret, error) {
-	if k.values.VPN.ReversedVPNEnabled {
-		return nil, nil
-	}
-
-	secret, err := k.secretsManager.Generate(ctx, &secretutils.VPNTLSAuthConfig{
-		Name: SecretNameVPNSeedTLSAuth,
```

Comment:
Can we now drop https://github.com/gardener/gardener/blob/44246e18b204925bc8fe782ef978ab1cdfa4066c/pkg/operation/botanist/component/kubeapiserver/deployment.go#L46?